pkgs/applications/misc/mkchromecast/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, buildPythonApplication, pulseaudio
+, PyChromecast, psutil, mutagen, flask, pyqt5, netifaces, requests, soco
+, vorbis-tools, sox, lame, flac, faac, ffmpeg }:
+buildPythonApplication rec {
+  pname = "mkchromecast";
+  version = "5872a246f0610b74fc2b197eb02dc91b96fb68cc";
```

Comment:
Should be something human readable, like "unstable-2011-12-31" (change the date to the commit date of course)